### ISSUE TITLE
feature request: master search

### ISSUE THREAD

**Issue by omeringen** [4gray/iptvnator#296]
We have to enter a category to search a content. I am requesting master search feature. Ability to search content on all categories. Search result might bring contents seperated by categories or something you decide better.

**Comment by 4gray** [comment]
Are you talking about search in xtream playlists?

**Comment by dobunzli** [comment]
Hello, global search would be a nice feature for stream playlists.

**Comment by omeringen** [comment]
> Are you talking about search in xtream playlists?

Yep. Many of app have this feature. I am talking about searching in just one xtream playlist. Most of time it's hard to know where to search in. (main categories or it's sub-directories)

Searching about ASDF for example.
Results might be like:

Live(2)
ASDFbla blaASDF

VOD(0)

Series(1)
ASDF
